src/pages/Explorer/Layouts/SheetsTemplate.svelte

```diff
@@ -0,0 +1,40 @@
+<script>
+    import Svg from 'webkit/ui/Svg/svelte'
+    export let item;
+    let isOpen = false;
+</script>
+
+<div class="column" on:click={() => isOpen = !isOpen}>
+    <div class="row justify v-center">
+        <h5 class="body-3">{item.title}</h5>
+        <Svg id='arrow' w="16" />
+    </div>
+    {#if isOpen}
+        <div class="row column mrg-l mrg--t mrg--b desc">
+            <p class="body-3 mrg-s mrg--b">
+                {item.desc}
+            </p>
+            <a href={item.url} target="_blank" class="row v-center c-white body-3">
```

Comment:
1. Primary buttons are created by using `btn-1` class;
2. This button has small padding. This can be achieved by adding `btn--s` class;
3. Same as the `[D-1-1]`

In result: `class="btn-1 btn--s row v-center"`